src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -879,9 +881,34 @@ def update_agents(cmd, client, resource_group_name, cluster_name, https_proxy=""
     # Get Helm chart path
     chart_path = utils.get_chart_path(registry_path, kube_config, kube_context, helm_client_location)
 
+    cmd_helm_values = [helm_client_location, "get", "values", "azure-arc", "--namespace", release_namespace]
+    if kube_config:
+        cmd_helm_values.extend(["--kubeconfig", kube_config])
+    if kube_context:
+        cmd_helm_values.extend(["--kube-context", kube_context])
+
+    if(operating_system == 'Windows'):
```

Comment:
Instead of doing this, use python apis which are generic for all OS
https://github.com/Azure/azure-cli-extensions/blob/284295daaf4e266fcad57bb997293bc74afb8b09/src/connectedk8s/azext_connectedk8s/_utils.py#L77

Reply:
Done. Thanks